utils/firestore.js

```diff
@@ -1237,6 +1237,100 @@ const searchSpecimen = async (masterSpecimenId, siteCode, allSitesFlag) => {
     return {};
 }
 
+/**
+ * getSiteLocationBox returns array of a single box object that matches the site and boxId
+ * @param {number} requestedSite - site code of the site
+ * @param {string} boxId - boxId of the box
+*/
+const getSiteLocationBox = async (requestedSite, boxId) => {
+    try {
+        const snapshot = await db.collection('boxes')
+                                .where(fieldMapping.loginSite.toString(), "==", requestedSite)
+                                .where(fieldMapping.shippingBoxId.toString(), "==", boxId).get();
+        const boxMatch = [];
+        for(let document of snapshot.docs) {
+            boxMatch.push(document.data());
+        }
+        return boxMatch;
+    } catch (error) {
+        throw new Error(`getSiteLocationBox() error: ${error.message}`);
+    }
+}
+
+/**
+ * getBiospecimenCollectionIdsFromBox returns an array of collectionIds from the box
+ * @param {number} requestedSite - site code of the site
+ * @param {string} boxId - boxId of the box
+*/
+const getBiospecimenCollectionIdsFromBox = async (requestedSite, boxId) => {
+    try {
+        const shipBoxMatch = await getSiteLocationBox(requestedSite, boxId);
+
+        if (shipBoxMatch === undefined || shipBoxMatch.length === 0) return [];
+        else {
```

Comment:
nit: same thing here. Since you're returning in the `if` the `else {}` isn't needed. We can just execute the logic without the `else {}`.

Reply:
Thanks for the suggestion.